core/src/test/scala/org/apache/spark/sql/delta/DeltaCDCSuite.scala

```diff
@@ -963,6 +963,30 @@ class DeltaCDCScalaSuite extends DeltaCDCSuiteBase {
     }
   }
 
+  test("Repeated delete") {
+    withTempDir { dir =>
+      val path = dir.getAbsolutePath
+      val deltaLog = DeltaLog.forTable(spark, path)
+      spark.range(1, 10, 1, 1).write.format("delta").save(path)
+      sql(s"delete from delta.`$path` where id = 3")
+      sql(s"delete from delta.`$path` where id = 4")
```

Comment:
```suggestion
      sql(s"DELETE FROM delta.`$path` WHERE id = 3")
      sql(s"DELETE FROM delta.`$path` WHERE id = 4")
```

Reply:
Use a file with 5 rows and include one more delete that will fully remove a file with a DV.

Reply:
Done!